app/interfaces/api/entities/cclf/final_claim.rb

```diff
@@ -7,10 +7,11 @@ def bills
           data = []
           data.push AdaptedFixedFee.represent(object.fixed_fee) if object.fixed_fee.present?
           data.push AdaptedGraduatedFee.represent(object.graduated_fee) if object.graduated_fee.present?
-          # data.push API::Entities::CCLF::AdaptedMiscFee.represent(miscellaneous_fees)
-          # data.push API::Entities::CCLF::AdaptedDisbursments.represent(disbursements)
+          data.push AdaptedMiscFee.represent(object.misc_fees)
```

Comment:
**[not a blocker]** Don't think this is a blocker for this PR, but I think this code could use with a bit of refactoring for DRY purposes 👍

Reply:
thanks, will try removing, there was a reason i had this in there, but don't recall and now the last line does some 'tidying up' anyway